test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1258,7 +1325,7 @@ static public void testCreateLargeObjectAndMegaRef() throws Throwable {
 	static public void testCreateLargeNumberOfPoint2D() throws Throwable {
 		String fields[] = {"x:I", "y:I"};
 		String className = "Point2D";
-		for(int valueIndex = 0; valueIndex < 200000; valueIndex++){
+		for(int valueIndex = 0; valueIndex < 2000; valueIndex++){
```

Comment:
why was this changed?

Reply:
I'm changing it back. This test was taking very long so I made it faster when debugging. Now it should be changed back.